db/migrations/20171006163153_create_posts_and_comments.cr

```diff
@@ -1,13 +1,13 @@
 class CreatePostsAndComments::V20171006163153 < Avram::Migrator::Migration::V1
   def migrate
     create :posts do
-      primary_key id : Int64
+      primary_key custom_id : Int64
       add_timestamps
       add title : String
     end
 
     create :comments do
-      primary_key id : Int64
+      primary_key custom_id : Int64
```

Comment:
Using custom keys in a number of relationships to make sure everything works fine